### ISSUE TITLE
separate label for defang/system related log messages

### ISSUE THREAD

**Issue by beorn** [DefangLabs/defang#255]
Build and CD logs are currently separated out by having separate labels (decker-app-image and cd) but some other system messages (e.g., the status=TASK_PENDING) have the same label as my app's logs, so it's impossible to filter them out.

```log
2024-04-05T22:32:17.463023Z ey15dt80q9g4 decker-app-image INFO[0051] ENTRYPOINT ["/run"]
2024-04-05T22:32:17.463090Z ey15dt80q9g4 decker-app-image INFO[0051] No files changed in this command, skipping snapshotting.
2024-04-05T22:32:20.920167Z ey15dt80q9g4 decker-app-image INFO[0054] Pushing image to 993944650415.dkr.ecr.us-west-2.amazonaws.com/kaniko-build:decker-app-image-94697038-x86_64
2024-04-05T22:32:23.802650Z ey15dt80q9g4 decker-app-image INFO[0057] Pushed 993944650415.dkr.ecr.us-west-2.amazonaws.com/kaniko-build@sha256:f05fe086d68500be1ab9a7c9c828c41cff86b5677d7a82bcd1b1c08ed4404e9e
2024-04-05T22:32:42.262065Z ey15dt80q9g4 cd Update completed in 1m45.003793669s ; provisioning...
2024-04-05T22:32:43.008622Z tn9qf51j0533 decker-app url="/" requesting
2024-04-05T22:32:54.948907Z ey15dt80q9g4 decker-app status=TASK_PROVISIONING
2024-04-05T22:33:04.697789Z ey15dt80q9g4 decker-app status=TASK_PENDING
2024-04-05T22:33:11.790731Z ey15dt80q9g4 decker-app status=TASK_PENDING
2024-04-05T22:33:12.952001Z tn9qf51j0533 decker-app [DEBUGGER] IN –> Awareness/Update
```

**Comment by lionello** [comment]
If you add `--verbose` there's an additional column "source" which would be an IP address for your service logs, but `fabric` for the status updates. It's not consistent with our image builder, which shows `decker-app-image` (but also shows `kaniko` for the source column)

**Comment by lionello** [comment]
Related to #69 , partly fixed by #275

**Comment by lionello** [comment]
Closing in favor of #426
